api/client_test.go

```diff
@@ -1,31 +1,139 @@
 package api
 
 import (
+	"context"
+	"log"
+	"net"
 	"testing"
 
+	"github.com/cedana/cedana/api/services/task"
+	"github.com/cedana/cedana/utils"
 	bolt "go.etcd.io/bbolt"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/test/bufconn"
 )
 
 func Test_Dump(t *testing.T) {
+}
+
+func TestClient_RunTask(t *testing.T) {
+	t.Run("TaskIsEmpty", func(t *testing.T) {
+		lis := bufconn.Listen(1024 * 1024)
```

Comment:
can you clean this up by consolidating the grpc specific code into another function and just calling them manually in each test?